src/app/filters/filter-fields.component.less

```diff
@@ -0,0 +1,47 @@
+@import (reference) "../../assets/stylesheets/patternfly-ng";
+
+.filter-pf {
+  a { cursor: pointer; }
+  &.filter-fields {
+    .form-group {
+      padding-left: 0;
+      width: 275px;
+    }
+    .btn-default { font-size: 12px; }
+    .typeahead-input-container {
+      position: relative;
+      padding-right: 0;
+      .caret {
+        position: absolute;
+        top: 10px;
+        right: 5px;
+        z-index: 2;
+      }
+    }
+  }
+}
+
+.filter-select {
+  .btn-default {
+    background-color: @color-pf-white; // #fff
+    background-image: none;
+    color: @color-pf-black-500; // #8b8d8f
+    font-size: 12px;
+    font-style: italic;
+    font-weight: 400;
+  }
+  .avatar {
+    height: 20px;
+    margin-right: 5px;
+  }
+}
+
+.input-group {
+  .input-group-btn {
+    .dropdown-menu>.selected>a {
+      background-color: @color-pf-blue !important; // #0088ce
+      border-color: #0076b7 !important;
```

Comment:
Again, this was taken verbatim from the Angular PF. I'm assuming there was a reason for doing that, but don't know what the issue was? Didn't want to change code without understanding it's history.

The color looks close to @color-pf-blue-400, so I'll use that instead.